package.json

```diff
@@ -49,6 +49,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.7",
+    "@babel/node": "^7.8.7",
```

Comment:
Why do we need babel/node? We don't need it for any of our other scripts.

Also: IMO this script does not need to be in package.json. Just putting it in `scripts/createLedger.js` and then wiring it into the backend build is fine.

Reply:
Followup: I guess the advantage to babel-node is that we don't need to manually wire it in as a backend entry point, unlike these?

https://github.com/sourcecred/sourcecred/blob/5b0faf284e53112581a21698b887b82767f07621/config/paths.js#L31-L43

I'm open to the idea that this is a better way to do things, but if so we should do it as a separate commit where we migrate all the scripts. For now can we just make this PR add a new backendEntryPoint?